pkg/integration/tests/interactive_rebase/drop_todo_commit_with_update_ref.go

```diff
@@ -9,7 +9,7 @@ var DropTodoCommitWithUpdateRef = NewIntegrationTest(NewIntegrationTestArgs{
 	Description:  "Drops a commit during interactive rebase when there is an update-ref in the git-rebase-todo file",
 	ExtraCmdArgs: "",
 	Skip:         false,
-	GitVersion:   From("2.38.0"),
+	GitVersion:   AtLeast("2.38.0"),
```

Comment:
@jesseduffield I know I convinced you to go with Before/From, but after seeing it at the usage sites I now think that `AtLeast` reads more naturally. I would still keep `Before` for the other one though.